nautobot_floor_plan/urls.py

```diff
@@ -4,14 +4,15 @@
 from django.urls import path
 from django.views.generic import RedirectView
 from nautobot.apps.urls import NautobotUIViewSetRouter
-from nautobot.extras.views import ObjectChangeLogView, ObjectNotesView
+from nautobot.core.views import ObjectChangeLogView, ObjectNotesView
```

Comment:
possibly `nautobot.apps.views`?

Reply:
But, I am not sure why these views are here in the first place. I think this might be a drift-manager issue.